onecgiar-pr-client/src/app/pages/results/pages/results-outlet/pages/results-notifications/components/update-notification/update-notification.component.html

```diff
@@ -0,0 +1,61 @@
+@let emitterName = notification?.obj_emitter_user?.first_name + ' ' + notification?.obj_emitter_user?.last_name; @let initOfficialCode =
+notification?.obj_result?.obj_result_by_initiatives[0]?.obj_initiative?.official_code; @let resultUrl = '/result/result-detail/' +
+notification?.obj_result?.result_code + '/general-information?phase=' + notification?.obj_result?.obj_version?.id;
```

Comment:
@xKeCo can the variable definitions be on their own lines?

Reply:
I'm using the new syntax they introduced in Angular 18. [@let Sintax](https://angular.dev/api/core/@let)